main.py

```diff
@@ -410,6 +411,9 @@ async def get_and_delete_old_versions(image_name: str, inputs: Inputs, http_clie
                 # Skipping, because the filter_include_untagged setting is False
                 continue
 
+            # If we got here, most probably we will delete image.
+            # For pseudo-branching we set delete_image to true and
+            # handle cases with delete image by tag filtering in separate pseudo-branch
```

Comment:
Could you explain this a bit more? What are pseudo-branches?

Reply:
In my understanding, this code looks a lot like if .. else.
So i've decided to make comment that helps understand the logic here.